Makefile

```diff
@@ -222,6 +222,14 @@ itest-coverage-data:
 	# exclude generated files from coverage data
 	grep -vE $(EXCLUDE_COVERAGE_FILES) $(TEST_OUTPUT)/itest-covdata.all.txt > $(TEST_OUTPUT)/itest-covdata.txt
 
+.PHONY: oats-test
+oats-test:
```

Comment:
Very cool, could we please add this Makefile target as part of the integration test target. Unless you are not confident this test works yet?

Reply:
Done, a 3rd PR action will now setup and run "make oats-test"